### DISCUSSION TITLE
Unable to install by Altstore

### DISCUSSION THREAD

**Discussion by ACEamword** [EhPanda-Team/EhPanda#309] [Q&A]
When the software installing via AltStore, it is unable to install the newest version or then the version which is newer than 2.4.0. The ver2.4.0 is able to install. Any ver above 2.4.0 won't be able to install. The AltStore shows the app is invalid.

**Comment by ACEamword** [comment]
I am using ios16.1

**Reply by Nebulosa-Cat** [reply]
if got this problem you can try when using altstore install don't install extension, just install with out the extension. 

The extension seems to add share botten at share page, not install it doesn't break the main function of EhPanda app.
